_posts/2025-05-12-using-root-for-genome-sequencing.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Using ROOT for genome sequencing"
+layout: post
+excerpt: "A GSoC 2025 project aiming to advance genomic data management by implementing ROOT's next-generation RNTuple format for sequence alignment storage."
+sitemap: false
+author: Aditya Pandey
+permalink: blogs/gsoc25_aditya_pandey_introduction_blog/
+banner_image: /images/blog/gsoc-banner.png
```

Comment:
Can we make the banner somehow more exciting. Some image on dna sequencing overlayed by the gsoc logo?